### ISSUE TITLE
When assigning tasks, Message All does not distribute an e-mail assignment to volunteers

### ISSUE THREAD

**Issue by OhMcGoo** [HTBox/allReady#343]
I began assigning tasks to volunteers (including myself). I checked the boxes next to the names of the three interested volunteers, clicked Save, clicked Message All and then sent a message (including instructions for next actions). No e-mail messages were received from the app.

![message all](https://cloud.githubusercontent.com/assets/4147733/11157460/1e2664ce-8a17-11e5-960b-43c9862a94f4.png)

**Comment by dpaquette** [comment]
Likely related to the queue-storage issues seen in #330

**Comment by tonysurma** [comment]
@dpaquette good point, I wonder if the errors are eaten and not seen.  I will check the sendgrid logs to verify if anything is sent

**Comment by tonysurma** [comment]
Verified, no emails made it to sendgrid

**Comment by MisterJames** [comment]
@OhMcGoo @tonysurma @dpaquette The code that would actually send out the notifications is currently commented out. I'm investigating and should have a patch tomorrow AM. Dave, you were right in that it's the same block of code currently under investigation in #330, but it doesn't get quite that far as the queue bits are not enabled.

**Comment by tonysurma** [comment]
Reopening as there are no webjobs to process the queues :cry: showing up in azure websites cc @MisterJames @dpaquette

**Comment by MisterJames** [comment]
@tonysurma  I think we can close this as it will resolve with #348...or at least close with that one.

**Comment by tonysurma** [comment]
Still don't have proof it is working, I suspect it is due to no volunteers with verified emails

**Comment by OhMcGoo** [comment]
As an Administrator, I attempted to Message All (I am one of the volunteers also) but have not received an e-mail.

<img width="706" alt="message all" src="https://cloud.githubusercontent.com/assets/4147733/11167544/0ddd1458-8b2c-11e5-900f-af2eac6b19ea.png">

**Comment by tonysurma** [comment]
Yep @OhMcGoo same here.  I am 99% sure the issue is the code (properly) only emails to users with verified emails (so you don't type someone else's email and spam them) and long story short we don't have users with verified emails.  I create a few issues to fix that (either allow us to have users reverify from account page so they can do it themselves if something doesn't work the first time (like the errors we have had) or allow us to delete users so they can try again with the email addresses on a new account and verify email addresses)

**Comment by tonysurma** [comment]
Confirmed this works! Houston we have contact!  I have been able to now test full cycle and it worked.  Once we have a confirmed email and text message the message all functionality worked.  Thank you everyone.  @OhMcGoo anyone who registered as a user needs to login again, click on their name in the top right and confirm their email - which requires hitting the link on that page and then copying and pasting the url from the email they receive into their web browser.

We have an issue #359 to make the email easier to click in the email
